### PR TITLE
Update readme, configs

### PR DESCRIPTION
**Pull request by aguestuser**
To expedite onboarding new devs (w/ bias toward debian), do the following:

* remove & gitignore `database.yml`
* add `database.yml.sample`
* extend dependency section(s)
* extend db setup section
* add instructions for running app locally
* add a few debian-friendly goodies! :)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,6 +11,9 @@
 /db/*.sqlite3
 /db/*.sqlite3-journal
 
+# Ignore files w/ secrets
+/config/database.yml
```
Comment (latazzajones) [inline .gitignore]: 👍 nice!

File: .gitignore [hunk 2]
```diff
@@ -21,3 +24,6 @@
 
 # Ignore Byebug History
 .byebug_history
+
+# Ignore Emacs stuff
+*~
```
Comment (latazzajones) [inline .gitignore]: I'm not opposed to this but wondering if it's something that can be put in your global git ignore instead?

Comment (aguestuser) [inline .gitignore]: PS: realize it's after the fact now, but i think it's a totally on-point suggestion to keep this out of the project `.gitignore` and leave editor-specific configs to those with said editor. also: hi @latazzajones! nice to meet you! :)PS: realize it's after the fact now, but i think it's a totally on-point suggestion to keep this out of the project `.gitignore` and leave editor-specific configs to those with said editor. also: hi @latazzajones! nice to meet you! :)

Comment (latazzajones) [inline .gitignore]: 👋

**Pull request merged**